### ISSUE TITLE
read.py produces inconsistent voices from sentance to sentance

### ISSUE THREAD

**Issue by embanot** [neonbjb/tortoise-tts#330]
when I run the read.py script, I'm noticing that the voice will alter a bit (sometimes quite noticable difference) between each sentance or individual clip.  Is there a way to specify that the voice be consistent through each of the clips generated which are then combined together?

**Comment by XOCODE-OP** [comment]
yes I would be interested too I wonder if this was answered somewhere.
How do generate a random voice but then hold on to that seed(random vector) to generate more of this particular random voice?

**Comment by XOCODE-OP** [comment]
nevermind its just --seed <int> in do_tts, doesnt say in the guide but its in the code
